backend/strandio.patch

```diff
@@ -0,0 +1,2 @@
+958d957
+<   ;<  ~      bind:m  (sleep ~s0)  ::  wait for thread to start
```

Comment:
As discussed, not clear why this sleep is here in the first place. Could be worth PRing to urbit/urbit, but also out of scope for here.